book/website/collaboration/ppie-events/ppie-events-tech.md

```diff
@@ -0,0 +1,100 @@
+This can help remind your attendees of their registration and help put the important information about your event at the top of their email inbox.
+
+It can also be helpful to utilize the subject line of your email with important information. 
+Consider including the date and title of your event, in both the calendar invitation and any email reminders that you send. 
+
+```{note}
+The Public and Patient Involvement and Engagement community for Multiple Long-Term Conditions has highlighted the importance of sending out reminder emails the week of the event, and the morning of the event. 
+
+This was because medical appointments had often been scheduled after an event's "Save the Date" had been sent and many community members valued the reminders. 
+```
+
+Ensure that any email address you send invitations from is able to contact your attendees (and not flagged as a spam email account). 
+You may suggest that attendees add your event's core email address to a list of trusted senders, or you can also ask for feedback from attendees leading up to an event, which allows you to identify if your emails have been received. 
+
+### Reaching attendees without email access
+Your event may engage communities and people with lived experience who have barriers to accessing email or the internet regularly. 
+
+Consider how you will keep in contact with these individuals, and what additional resources and safeguarding your organizers and staff members may need. 
+For example, if attendees cannot access email, what other communication channels can be used. 
+If you are able to send out information and keep in regular contact with the community through the telephone, social media, or a group chat, consider devoting resources towards a "work phone" for organizers so that they do not need to give out their personal phone number to attendees.
+
+If this is the case for your event, it will be very important to source solutions from, and work alongside the community with that specific lived experience when planning your event. 
+
+
+(cl-ppie-events-technology-guides)=
+## Technology Guides
+Even if the technology used during your event may be considered common or ubiquitous, it is important to recognize that all attendees will have had a different level of experience and exposure to technology. 
+
+While many people with lived experience may be experts in the latest technology, this may not be the case for all attendees. 
+
+Consider developing and providing tutorials and guides for each different technology tool that your attendees will interact with at your event. 
+This is important for both in-person and especially so for online or hybrid events. 
+
+This includes guides for technology such as [Zoom](https://zoom.us/), [Slido](https://www.slido.com), [Slack](https://slack.com/), and any interactive software such as Miro or Microsoft Whiteboards. 
```

Comment:
Oh, I know I added a comment above about Slido links. They're here! Maybe having them linked in both places could work well?